### PR TITLE
update documentation and config for external tor

### PR DESCRIPTION
**Pull request by woodser**
Replaces https://github.com/haveno-dex/haveno/pull/1170 and https://github.com/haveno-dex/haveno/pull/1182

### CODE REVIEW

**Comment by woodser** [comment]
@shortwavesurfer2009 @boldsuck @preland

I've reposted the documentation and config changes for using external tor. Can you please review it for final changes?

File: seednode/torrc [hunk 1]
```diff
@@ -1,18 +1,144 @@
-RunAsDaemon 1
```
Comment (woodser) [inline seednode/torrc:1]: Should this be present in the new tor config?

**Comment by preland** [comment]
It looks good to me; I'll wait and see what they have to say.

**Comment by shortwavesurfer2009** [comment]
No longer needed i think since 1 above builds it
3. Rebuild the seed node `./gradlew build -x test`.

If 3 removed renumber list

If 3 is removed it would be step 4
Replace `<user>` above with the user that will be running the seed node (step 6 above)

If current step 3 removed the 5 becomes 3
11. Add the `.onion` address from step 5 to `core/src/main/resources/xmr_<network>.seednode

File: seednode/torrc (hunk 1 above)
Comment (boldsuck) [reply seednode/torrc:1]: This is done by `/usr/share/tor/tor-service-defaults-torrc
`
```
DataDirectory /var/lib/tor
PidFile /run/tor/tor.pid
RunAsDaemon 1
User debian-tor

ControlSocket /run/tor/control GroupWritable RelaxDirModeCheck
ControlSocketsGroupWritable 1
SocksPort unix:/run/tor/socks WorldWritable
SocksPort 9050

CookieAuthentication 1
CookieAuthFileGroupReadable 1
CookieAuthFile /run/tor/control.authcookie

Log notice syslog

```

Users who have multiple tor-instances configured by `tor-service-defaults-torrc-instances` need
`RunAsDaemon 0`

therefore I did not put it in the seednode torrc.

File: seednode/haveno-seednode.service
Comment (boldsuck) [inline seednode/haveno-seednode.service:1]: --appName=haveno-XMR_STAGENET_Seed_2002\
missing '\'

**Comment by woodser** [comment]
Replaced by #1186

**Pull request closed without merging**